lib/index.js

```diff
@@ -33,6 +33,25 @@ function getModuleIdentifier(seen, modulePath) {
   return `__${identifier}__`;
 }
 
+function shouldIgnore(pathParts) {
+  // ignore any of these extensions (.md, .html, etc)
+  if (IGNORED_EXTENSIONS.indexOf(pathParts.ext) > -1) {
+    return true;
+  }
+
+  // ignore .d.ts files
+  if (pathParts.base.slice(-5) === '.d.ts') {
+    return true;
+  }
+
+  // ignore dotfiles
+  if (pathParts.base[0] === '.') {
```

Comment:
Ya, I agree.

We can refactor to make `IGNORED_PREFIXES` and `IGNORED_SUFFIXES` and then use something like `IGNORED_PREFIXES.some(prefix => pathParts.base.startsWith(prefix))`